scripts/release/controller/socket.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+# TODO: ssh-keyscan?
+# -o StrictHostKeyChecking=no suppresses the (yes/no) new key ssh question.
+# This lessens the security, but it may be acceptable in this case.
+
+INSTANCE=$1
```

Comment:
Test the variable; if it's empty fail the script.

Reply:
Will do.